### PR TITLE
What's Here for ENV

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: hash.c [hunk 1]
```diff
@@ -7312,6 +7312,85 @@ Init_Hash(void)
      * so most example snippets begin by resetting the contents of ENV:
      * - ENV.replace replaces ENV with a new collection of entries.
      * - ENV.clear empties ENV.
+     *
+     * == What's Here
+     *
+     * First, what's elsewhere. \Class \ENV:
+     *
+     * - Inherits from {class Object}[Object.html#class-Object-label-What-27s+Here].
+     *
```
Comment (jeremyevans) [inline hash.c:7321]: Here is where you would usually list that `ENV` extends `Enumerable`, I think.

**Review by jeremyevans — APPROVED** [review]
Looks good.  Up to you if you want to mention that `ENV` extends `Enumerable`, but I would recommend doing so.  No need for further review.

**Comment by nobu** [comment]
Could you prefix the commit message with `[DOC]` at merging?

**Comment by nobu** [comment]
I meant the top of the whole message including the first line, or the title of the pull request.

**Pull request merged**